en/docs/develop/restapis/keystore.yaml

```diff
@@ -0,0 +1,337 @@
+openapi: 3.0.0
+info:
+  description: This is the REST API for Keystore related operations in WSO2 Identity Server
+  version: "1.0.0-oas3"
+  title: WSO2 Identity Server - Keystore API
+  contact:
+    name: WSO2 Identity Server
+    url: 'https://wso2.com/identity-and-access-management'
+    email: architecture@wso2.org
+  license:
+    name: Apache 2.0
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+servers:
+  - url: 'https://apis.is.com/t/{tenant-domain}/api/server/v1'
```

Comment:
Change the URL to "https://localhost:9443/t/carbon.super/api/server/v1", so that all the URLs generated by swagger-ui are well constructed.

Reply:
fixed